File: beecrowd/python/ad-hoc/ex1536.py
n = int(input())

for i in range(n):
    m1, v1 = map(int, input().split('x'))
    v2, m2 = map(int, input().split('x'))
    
    if (m1+m2) > (v1+v2):
        print('Time 1')
    elif (m1+m2) < (v1+v2):
        print('Time 2')
    else:
        if v1 > m2:
            print('Time 2')
        elif v1 < m2:
            print('Time 1')
        else:
            print('Penaltis')

File: beecrowd/c++/ad-hoc/ex1536.cpp
#include <iostream>
#include <cstdio>

using namespace std;

int main(){
    int n, m, v, m1, v1, v2, m2;
    char x;
    cin>>n;
    
    for (int i=0; i<n; i++){
        cin>>m1>>x>>v1;
        cin>>v2>>x>>m2;
        m = m1 + m2;
        v = v1 + v2;
        if (m>v){
            cout<<"Time 1"<<endl;
        } else if(m<v){
            cout<<"Time 2"<<endl;
        } else{
            if (m2>v1){
                cout<<"Time 1"<<endl;
            } else if (m2<v1){
                cout<<"Time 2"<<endl;
            } else{
                cout<<"Penaltis"<<endl;
            }
            
        }
    }
    
    return 0;
}
